cmake/AddInputFileTests.cmake

```diff
@@ -47,6 +47,15 @@ function(add_single_input_file_test INPUT_FILE EXECUTABLE CHECK_TYPE TIMEOUT)
       # Make sure we run the test in the build directory for cleaning its output
       WORKING_DIRECTORY ${CMAKE_BINARY_DIR}
       )
+  elseif("${CHECK_TYPE}" STREQUAL "execute_check_h5")
```

Comment:
we should add to the `Having your Contributions Merged into SpECTRE` section a `Testing Input Files` section. Would you be willing to either write this documentation or file an issue so we remember to add it (please mark the issue with the `documentation` label)?

Reply:
Sure, I can write that documentation.

Reply:
Opened #2609 -- I'll do it in another PR to avoid further cluttering this one.